impc_api_helper/impc_api_helper/solr_request.py

```diff
@@ -23,6 +23,20 @@ def solr_request(core, params, silent=False):
             'fl': 'marker_symbol,allele_symbol,parameter_stable_id',  # Fields to retrieve
         }
     )
+    
+    Faceting query provides a summary of data distribution across the specified fields.
+    Example faceting query:
+        num_found, df = solr_request(
+        core='genotype-phenotype',
+        params={
+            'q': '*:*',
+            'rows': 0,
+            'facet': 'on',
+            'facet.field': 'zygosity',
+            'facet.limit': 15,
+            'facet.mincount': 1
```

Comment:
It might be beneficial to document what each param does for users the documentation. 

Something like: 

```suggestion
            'q': '*:*',
            'rows': 0,
            'facet': 'on',        # enables  facet request
            'facet.field': 'zygosity', 
            'facet.limit': 15,    # number of results returned 
            'facet.mincount': 1 # ...
```

Reply:
Wonderful idea! I will create issue to implement this later

Reply:
Issue #2